index.html

```diff
@@ -67,6 +67,19 @@
   </head>
 
   <body>
+    <div class="dropdown hide">
+      <ul class="dropdown-list">
+        <li class="dropdown-item">
+          <a class="dropdown-link" href="https://my.realdevsquad.com/"
+            >My Profile</a
+          >
```

Comment:
Can we please fix the formatting ?

Reply:
```suggestion
<a class="dropdown-link" href="https://my.realdevsquad.com/">
  My Profile
</a>
```
OR
```suggestion
<a class="dropdown-link" href="https://my.realdevsquad.com/">My Profile</a>
```

Reply:
That is auto-formatting from npm run fix without that the lint checks will fail